v1.1/cancel-job.md

```diff
@@ -0,0 +1,49 @@
+---
+title: CANCEL JOB
+summary: The CANCEL JOB statement stops long-running jobs, which include enterprise BACKUP and RESTORE.
+toc: false
+---
+
+<span class="version-tag">New in v1.1</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
```

Comment:
nit: add colon to new callout `New in v1.1:`

Reply:
Done.